examples/version-migration/same-container/tests/sameContainer.test.ts

```diff
@@ -3,9 +3,9 @@
  * Licensed under the MIT License.
  */
 
-import { IMigrator } from "@fluid-example/example-utils";
+import type { ISameContainerMigrator } from "@fluid-example/example-utils";
```

Comment:
In the process realized we were using the wrong interface in a few places for the same-container flow (but weren't getting bitten because they were similar enough).